R/calc.dtsurvey.R

```diff
@@ -0,0 +1,364 @@
+#' Compute CHI type estimates from a dtsurvey object
+#' @rdname calc
+#' @export
+#' @importFrom data.table substitute2
+#' @importFrom stats median na.omit
+calc.dtsurvey = function(ph.data,
+                         what,
+                         where, #this is a change from the main calc framework
+                         by = NULL,
+                         metrics = c('mean', 'numerator', 'denominator'),
+                         per = NULL,
+                         win = NULL,
+                         time_var = NULL,
+                         proportion = FALSE,
+                         fancy_time = TRUE,
+                         ci = .95,
+                         verbose = FALSE,
+                         ...){
+
+  #filter the dataset
+  if(!missing(where)){
```

Comment:
This doesn't work on my computer. Says 'where' does not resolve to a logical
Think I had the same problem when I tried to address your concern re: eval(parse(text= in other code.

Reply:
What specifically is the command that is breaking?

Reply:
My mistake! It does work. I think I tried it with quotes and it tanked.